benchmark/bench_BQRRP/BQRRP_speed_comparisons_block_size.cc

```diff
@@ -59,12 +59,17 @@ static void call_all_algs(
     int64_t numruns,
     int64_t b_sz,
     QR_speed_benchmark_data<T> &all_data,
+    std::string operation_mode,
```

Comment:
Please document the intended possible values of ``operation_mode``.

Reply:
Fixed